README.md

```diff
@@ -1,6 +1,6 @@
-[![CircleCI](https://circleci.com/gh/giantswarm/kubernetes-coredns/tree/master.svg?style=svg)](https://circleci.com/gh/giantswarm/kubernetes-coredns/tree/master)
+[![CircleCI](https://circleci.com/gh/giantswarm/coredns-app.svg?style=svg)](https://circleci.com/gh/giantswarm/coredns-app)
```

Comment:
```suggestion
[![CircleCI](https://circleci.com/gh/giantswarm/coredns-app.svg?style=shield)](https://circleci.com/gh/giantswarm/coredns-app)
```

Readme should be aligned with other apps. See https://github.com/giantswarm/kube-state-metrics-app.